### PR TITLE
The Field Studies map no longer renders hidden result sets

### PR DESCRIPTION
**Pull request by zednis**
The result sets are not shown so we should not use a template to render them.

### CODE REVIEW

File: html/field-studies.html [hunk 1]
```diff
@@ -129,7 +129,8 @@
                 ],
                 render_result_record: function(options, record)
                 {
-                    return template(record).trim();
```
Comment (mrpatrickwest) [inline html/field-studies.html:132]: would it make sense to empty out all that code as well? If we need it back we can always get it from an earlier revision. So remove lines 24 to 66

Comment (zednis) [inline html/field-studies.html:132]: That would make sense.  We could also re-use the template for the pop-up HTML.

**Comment by zednis** [comment]
@mrpatrickwest I removed the unused result set template

**Comment by mrpatrickwest** [comment]
:shipit:

**Pull request merged**